src/openMVG/geometry/Similarity3.cpp

```diff
@@ -37,7 +37,7 @@ Pose3 Similarity3::operator () ( const Pose3 & pose ) const
 
 Similarity3 Similarity3::inverse() const
 {
-  return Similarity3(pose_.inverse(), 1.0 / scale_);
+  return Similarity3(Pose3(pose_.rotation(),-pose_.rotation()*pose_.center()*scale_), 1.0 / scale_);
```

Comment:
rotation needs to be inverted

Reply:
true, with the rotation inverted it solves #1285

Reply:
@Yannnnnnnnnnnn  @sebastienchappuis Do you want to make a PR with a unit test and the fix?

Reply:
made a stupid error here.

Reply:
@pmoulon I will try to add an unit test here and fix it.